apps/experiments/views/experiment.py

```diff
@@ -363,6 +363,19 @@ def extra_context(self):
             team_participant_identifiers.extend(self.object.participant_allowlist)
             team_participant_identifiers = set(team_participant_identifiers)
 
+        excluded_fields = [
+            "version_number",
+            "is_default_version",
+            "public_id",
+            "version_description",
+        ] + self.object.get_fields_to_exclude()
+        fields_changed = self.object.latest_version.compare_with_model(self.object, excluded_fields)
+        if "consent_form" in fields_changed:
```

Comment:
also offlined, but here is the (temp) fix for the consent forms being matched by id (which is different version to version always)